source/core/Console.cpp

```diff
@@ -247,12 +255,18 @@ namespace GTA
 		}
 
 		//We can't get the n-th, so let's do it with a counter
+		//page = 1 --> start: 0 end: 15
+		//page = 2 --> start: 16 end: 31
+		int start = (_page - 1) * 16; //0
+		int end = Math::Min(_lines->Count, _page * 16 - 1); //12
```

Comment:
Is there not a way to do scrolling by say only a couple of line, skipping down the whole page seems a bit much